### DISCUSSION TITLE
Retrieving current rollout percentage from Play Store

### DISCUSSION THREAD

**Discussion by benlaverriere** [fastlane/fastlane#19432] [Ideas]
🔑

I'd love to include our app's current (production) Android rollout percentage in _fastlane_'s output when we're modifying the rollout. As far as I can tell, this isn't something that `supply` or other actions expose currently. I think that the AndroidPublisher API client [can provide this number](https://github.com/googleapis/google-api-ruby-client/blob/596ff6cc93d4123d32665c2ba68d676aeea04fd9/generated/google-apis-androidpublisher_v3/lib/google/apis/androidpublisher_v3/classes.rb#L1970), and I'd be happy to work on a PR adding this capability — but I wanted to discuss the approach before starting. I imagine this could be a small, single-purpose action like `google_play_track_version_codes` and `google_play_track_release_names`, unless a plugin would be more appropriate? We'd need a similar set of params (auth, target URL, track, etc.).

(And if I've missed a way to get this value that already exists, please let me know!)

**Comment by Felix362** [comment]
Did you have a fix on this issue? Facing the same issue but no response from anyone and couldn't find this topic troubleshooting in google.
[PrepaidCardStatus](https://www.prepaidcardstatus.org/)

**Comment by Kardelio** [comment]
Also would love this feature, do you know if its possible now? the last comment was like 2 years ago... haha 🤞

**Comment by brainbicycle** [comment]
i implemented a version of this a while back but has not been reviewed: https://github.com/fastlane/fastlane/pull/22029
We use this via monkey patching for automating our rollouts: https://github.com/artsy/eigen/pull/10244/changes
